include/boost/geometry/algorithms/detail/overlay/cluster_info.hpp

```diff
@@ -27,11 +27,11 @@ struct cluster_info
     std::set<signed_size_type> turn_indices;
 
     //! Number of open spaces (e.g. 2 for touch)
-    std::size_t open_count;
+    std::size_t open_count{0};
 
-    inline cluster_info()
-        : open_count(0)
-    {}
+    //! Number of spikes, where a segment goes to the cluster point
+    //! and leaves immediately in the opposite direction.
+    std::size_t spike_count{0};
```

Comment:
This (and related code) is also necessary, to have a way out from a cluster which is closed, but where still is a way in and out. This is also related to floating point issues - the part between the spike legs might be indetectable